xrayvision/imaging.py

```diff
@@ -111,27 +111,29 @@ def image_to_vis(
         Array of u coordinates where the visibilities will be evaluated
     v :
         Array of v coordinates where the visibilities will be evaluated
-    phase_centre :
-        The coordinates the phase_centre.
+    phase_center :
+        The coordinates the phase_center.
     pixel_size :
         Size of pixels, if only one value is passed, assume square pixels (repeating the value).
 
     Returns
     -------
     :
-        The new visibility object
+        The new Visibilities object
 
     """
     pixel_size = validate_and_expand_kwarg(pixel_size, "pixel_size")
     if not (apu.get_physical_type((1 / u).unit) == ANGLE and apu.get_physical_type((1 / v).unit) == ANGLE):
         raise ValueError("u and v must be inverse angle (e.g. 1/deg or 1/arcsec")
-    vis = dft_map(image, u=u, v=v, phase_centre=phase_centre, pixel_size=pixel_size)
-    return Visibility(vis, u=u, v=v, offset=phase_centre)
+    vis = dft_map(
+        image, u=u, v=v, phase_center=[0.0, 0.0] * apu.arcsec, pixel_size=pixel_size
```

Comment:
Maybe this should be `offset` e.g. we want to create a vis with an offset of x and the `phase_center` is always 0,0 because the visibilities are created that way?

Reply:
See [here](https://github.com/TCDSolar/xrayvision/pull/67#discussion_r1622904240).